src/main/java/application/Handlers.java

```diff
@@ -0,0 +1,11 @@
+package main.java.application;
+
+import main.java.server.request.Request;
+
+class Handlers {
+    static final Handler SimpleGet = (Request request) -> HandlerHelper.assembleResponse(request, "");
+    static final Handler GetWithBody = (Request request) -> HandlerHelper.assembleResponse(request, "");
+    static final Handler SimpleOptions = (Request request) -> HandlerHelper.assembleResponse(request, "");
+    static final Handler EchoBody = (Request request) -> HandlerHelper.assembleResponse(request, "Request body");
```

Comment:
What is the `body` string doing here? Does every invocation of the `EchoBody` handler return `"Request body"`?

Reply:
`"Request body"` was a placeholder (that I mistakenly left in) because I hadn't built `request.body` yet, my mistake.